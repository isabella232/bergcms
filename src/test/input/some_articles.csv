2	0:Prolog (Vorlauf)	10	-Dokumentenvorlage	F	 	% Finale Version:<br>\documentclass{berg}<br>%<br>% Ohne Bilder, <br>% aber mit schwarzen Balken, die zu breite Bereiche kennzeichnen, die dann in anderes hineinragen <br>%\documentclass[draft]{berg}<br><br>%<br>\hypersetup{<br>    %bookmarks=true,          % show bookmarks bar? - Warning: `bookmarks' has already been used,<br>    unicode=false,            % non-Latin characters in Acrobat’s bookmarks<br>    pdftoolbar=true,          % show Acrobat’s toolbar?<br>    pdfmenubar=true,          % show Acrobat’s menu?<br>    pdffitwindow=false,       % window fit to page when opened<br>    pdfstartview={FitH},      % fits the width of the page to the window<br>    pdftitle={Berg CMS}, % title<br>    pdfauthor={See History of feginfo.csv},   % author<br>    pdfsubject={Ausgabe <br>>AUSGABEZEITRAUM#<br>},   % subject of the document<br>    %pdfcreator={Creator},    % creator of the document<br>    %pdfproducer={}, % producer of the document<br>    %pdfkeywords={keyword1} {key2} {key3}, % list of keywords<br>    pdfnewwindow=true,        % links in new window<br>    colorlinks=true,          % false: boxed links; true: colored links<br>    linkcolor=black,          % color of internal links<br>    citecolor=black,          % color of links to bibliography<br>    filecolor=black,          % color of file links<br>    urlcolor=black            % color of external links<br>}	%<br>\begin{document}<br>%	20130805-153002-Mo / 127.0.0.1:47571
3	0:Prolog (Vorlauf)	08	-Ausgabe festlegen	K	 	>DATEN#5#2013#500#10# Ausgabe festlegen: Monat#Jahr#Auflage#Redaktionsschluss(Tag)	 	20130805-150216-Mo / 127.0.0.1:47548
6	0:Prolog (Vorlauf)	40	-Inhaltsverzeichnis	F	%***************************************<br>%* Inhaltverzeichnis                   *<br>%***************************************	>sg#4#<br>>SPALTEN#2#<br>%>zd#1.5#<br>>zd#1.1#<br>>INHALT<br><br>>zd#0# Standardzeilendichte zurücksetzen	 	20130805-153105-Mo / 127.0.0.1:47573
13	0:Prolog (Vorlauf)	30	-Titelseite	F	%------- Titelseite --------------DIN A4---------------------------<br>\newlength{\boxwidth} \setlength{\boxwidth}{\textwidth}<br>\addtolength{\boxwidth}{-2\fboxsep}<br>\addtolength{\boxwidth}{-2\fboxrule}<br>\newlength{\boxheight} \setlength{\boxheight}{0.94\textheight}<br>\newlength{\indentheader} \setlength{\indentheader}{2.8mm}%<br>\addtolength{\boxwidth}{-\indentheader}%<br>\thispagestyle{empty}<br>%BigBox-------- Titelseite ohne overfull hbox-------DIN A4-------<br>\fbox{\hspace{\indentheader}\parbox[t][\boxheight]{\boxwidth}{%<br>{\bfseries <br>%<br>\begin{minipage}[b]{70mm}%<br>\Huge <br>Example Organization\\[-8mm]<br>\end{minipage}%<br>%\hfill\includegraphics[scale=0.21]{berg.jpg}%<br>\hspace{10mm}\vspace*{2mm}\\<br>\Large<br>\resizebox{175mm}{4.9mm}{using Berg CMS}<br>}%<br>\\%\vspace*{0.2ex}\\% Mindestabstand 0.5ex	%------- Titelbild -----------------------<br>\vspace*{5mm}% zusätzlicher Abstand<br><br>%>BILD#titel#c#datei#b:170mm#Fotograf<br>% oder Höhe: h:195mm<br>>bild##c#berg#b:100mm#Christian Leutloff<br><br>%	%------- Abschluss der Titelseite -------<br><br>\vfill<br>\vspace*{0.8ex}% Mindestabstand 0.8ex<br>}} %parbox fbox<br><br>\normalsize<br><br>>AUSGABE#<br>% Ende der Titelseite	20130805-151651-Mo / 127.0.0.1:47553
11	1:Gemeindeleben	020	-Gottesdienste	F	>1#Gottesdienste	%Datum  # Predigt  # Liturgie  # Besonderheit       # Musik       # Kollekte<br>%14mm  # 15mm       # 17mm          # 26mm              # 16mm         # 18mm<br>%14mm  # 14mm       # 16mm          # 26mm              # 20mm         # 16mm<br>%15mm  # 14mm       # 17mm          # 23mm              # 16mm         # 19mm<br>% war 17<br>>TABELLE#[b]#|#emph<br>15mm  # 14mm       # 17mm          # 21mm              # 18mm         # 19mm<br>Datum       # Predigt      # Liturgie     # Besonderheit           # Musik           # Kollekte<br>06.01. 10:15#A. Name#A. Name# # #Purpose<br>13.01. 10:15#A. Name #A. Name #Abendmahl#A. Name#Gemeinde<br>>*	>+#New Page	20130805-152032-Mo / 127.0.0.1:47557
14	1:Gemeindeleben	340	Arbeitssamstag	A		Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.<br><br>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.	 	20130805-152941-Mo / 127.0.0.1:47570
15	1:Gemeindeleben	320	Forumsabend: Zu einem bestimmten Thema	A		Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.<br><br>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.	 	20130805-152933-Mo / 127.0.0.1:47569
16	1:Gemeindeleben	100	-GEMEINDELEBEN - Ueberschrift	F	>1#Gemeindeleben	 	 	20130805-152429-Mo / 127.0.0.1:47566
