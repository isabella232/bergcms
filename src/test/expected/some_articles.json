Content-type: application/json; charset=utf-8

{
    "articles": [
        {
            "id": "2",
            "priority": "10",
            "type": "F",
            "chapter": "0:Prolog (Vorlauf)",
            "title": "-Dokumentenvorlage",
            "header": " ",
            "body": "% Finale Version:\n\\documentclass{berg}\n%\n% Ohne Bilder, \n% aber mit schwarzen Balken, die zu breite Bereiche kennzeichnen, die dann in anderes hineinragen \n%\\documentclass[draft]{berg}\n\n%\n\\hypersetup{\n    %bookmarks=true,          % show bookmarks bar? - Warning: `bookmarks' has already been used,\n    unicode=false,            % non-Latin characters in Acrobat’s bookmarks\n    pdftoolbar=true,          % show Acrobat’s toolbar?\n    pdfmenubar=true,          % show Acrobat’s menu?\n    pdffitwindow=false,       % window fit to page when opened\n    pdfstartview={FitH},      % fits the width of the page to the window\n    pdftitle={Berg CMS}, % title\n    pdfauthor={See History of feginfo.csv},   % author\n    pdfsubject={Ausgabe \n>AUSGABEZEITRAUM#\n},   % subject of the document\n    %pdfcreator={Creator},    % creator of the document\n    %pdfproducer={}, % producer of the document\n    %pdfkeywords={keyword1} {key2} {key3}, % list of keywords\n    pdfnewwindow=true,        % links in new window\n    colorlinks=true,          % false: boxed links; true: colored links\n    linkcolor=black,          % color of internal links\n    citecolor=black,          % color of links to bibliography\n    filecolor=black,          % color of file links\n    urlcolor=black            % color of external links\n}",
            "footer": "%\n%",
            "lastChanged": "20130805-153002-Mo \/ 127.0.0.1:47571 \/ bergcms"
        },
        {
            "id": "3",
            "priority": "8",
            "type": "K",
            "chapter": "0:Prolog (Vorlauf)",
            "title": "-Ausgabe festlegen",
            "header": " ",
            "body": ">DATEN#5#2013#500#10# Ausgabe festlegen: Monat#Jahr#Auflage#Redaktionsschluss(Tag)",
            "footer": " ",
            "lastChanged": "20130805-150216-Mo \/ 127.0.0.1:47548 \/ bergcms"
        },
        {
            "id": "6",
            "priority": "40",
            "type": "F",
            "chapter": "1:Einstieg",
            "title": "-Inhaltsverzeichnis",
            "header": "%***************************************\n%* Inhaltverzeichnis                   *\n%***************************************",
            "body": ">sg#4#\n>SPALTEN#2#\n%>zd#1.5#\n>zd#1.1#\n>INHALT\n\n>zd#0# Standardzeilendichte zurücksetzen",
            "footer": " ",
            "lastChanged": "20130805-153105-Mo \/ 127.0.0.1:47573 \/ bergcms"
        },
        {
            "id": "13",
            "priority": "30",
            "type": "F",
            "chapter": "1:Einstieg",
            "title": "-Titelseite",
            "header": "%------- Titelseite --------------DIN A4---------------------------\n\\newlength{\\boxwidth} \\setlength{\\boxwidth}{\\textwidth}\n\\addtolength{\\boxwidth}{-2\\fboxsep}\n\\addtolength{\\boxwidth}{-2\\fboxrule}\n\\newlength{\\boxheight} \\setlength{\\boxheight}{0.94\\textheight}\n\\newlength{\\indentheader} \\setlength{\\indentheader}{2.8mm}%\n\\addtolength{\\boxwidth}{-\\indentheader}%\n\\thispagestyle{empty}\n%BigBox-------- Titelseite ohne overfull hbox-------DIN A4-------\n\\fbox{\\hspace{\\indentheader}\\parbox[t][\\boxheight]{\\boxwidth}{%\n{\\bfseries \n%\n\\begin{minipage}[b]{70mm}%\n\\Huge \nExample Organization\\\\[-8mm]\n\\end{minipage}%\n%\\hfill\\includegraphics[scale=0.21]{berg.jpg}%\n\\hspace{10mm}\\vspace*{2mm}\\\\\n\\Large\n\\resizebox{175mm}{4.9mm}{using Berg CMS}\n}%\n\\\\%\\vspace*{0.2ex}\\\\% Mindestabstand 0.5ex",
            "body": "%------- Titelbild -----------------------\n\\vspace*{5mm}% zusätzlicher Abstand\n\n%>BILD#titel#c#datei#b:170mm#Fotograf\n% oder Höhe: h:195mm\n>bild##c#berg#b:100mm#Christian Leutloff\n\n%",
            "footer": "%------- Abschluss der Titelseite -------\n\n\\vfill\n\\vspace*{0.8ex}% Mindestabstand 0.8ex\n}} %parbox fbox\n\n\\normalsize\n\n>AUSGABE#\n% Ende der Titelseite",
            "lastChanged": "20130805-151651-Mo \/ 127.0.0.1:47553 \/ bergcms"
        },
        {
            "id": "11",
            "priority": "20",
            "type": "F",
            "chapter": "2:Gemeindeleben",
            "title": "-Gottesdienste",
            "header": ">1#Gottesdienste",
            "body": "%Datum  # Predigt  # Liturgie  # Besonderheit       # Musik       # Kollekte\n%14mm  # 15mm       # 17mm          # 26mm              # 16mm         # 18mm\n%14mm  # 14mm       # 16mm          # 26mm              # 20mm         # 16mm\n%15mm  # 14mm       # 17mm          # 24mm              # 16mm         # 19mm\n% war 17\n>TABELLE#[b]#|#emph\n16mm  # 14mm       # 17mm          # 21mm              # 18mm         # 19mm\nDatum       # Predigt      # Liturgie     # Besonderheit           # Musik           # Kollekte\n06.01. 10:15#A. Name#A. Name# # #Purpose\n13.01. 10:15#A. Name #A. Name #Abendmahl#A. Name#Gemeinde\n>*",
            "footer": ">+#New Page",
            "lastChanged": "20130805-152032-Mo \/ 127.0.0.1:47557 \/ bergcms"
        },
        {
            "id": "14",
            "priority": "340",
            "type": "A",
            "chapter": "2:Gemeindeleben",
            "title": "Arbeitssamstag",
            "header": "",
            "body": "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.\n\nLorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.",
            "footer": " ",
            "lastChanged": "20160805-152941-Mo \/ 127.0.0.1:47570 \/ hero"
        },
        {
            "id": "15",
            "priority": "320",
            "type": "A",
            "chapter": "2:Gemeindeleben",
            "title": "Forumsabend: Zu einem bestimmten Thema",
            "header": "",
            "body": "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.\n\nLorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.",
            "footer": " ",
            "lastChanged": "20130805-152933-Mo \/ 127.0.0.1:47569 \/ superguru"
        },
        {
            "id": "16",
            "priority": "100",
            "type": "F",
            "chapter": "2:Gemeindeleben",
            "title": "-GEMEINDELEBEN - Überschrift",
            "header": ">1#Gemeindeleben",
            "body": " ",
            "footer": " ",
            "lastChanged": "20130805-152429-Mo \/ 127.0.0.1:47566 \/ bergcms"
        }
    ]
}
